src/main/java/org/assertj/core/api/Assertions.java

```diff
@@ -1211,6 +1214,89 @@ public static String contentOf(File file) {
   }
 
   // --------------------------------------------------------------------------------------------------
+  // URL/Resource methods : not assertions but here to have a single entry point to all AssertJ features.
+  // --------------------------------------------------------------------------------------------------
+
+  /**
+   * Loads the text content of a URL, so that it can be passed to {@link #assertThat(String)}. <p> Note that this will
+   * load the entire file in memory. </p>
+   *
+   * @param url the URL.
+   * @param charset the character set to use.
+   * @return the content of the file.
+   * @throws NullPointerException if the given charset is {@code null}.
+   * @throws FilesException       if an I/O exception occurs.
```

Comment:
let's rename `FilesException` to `RuntimeIOException`, URL are not only about file.